src/app/cube/shared/footer/footer.component.html

```diff
@@ -13,7 +13,10 @@ <h3>Platform</h3>
         <h3>Help</h3>
         <a aria-label="Clickable Ask for Help link" href="mailto:info@secured.team?subject=CLARK Help">{{ copy.HELP }}</a>
         <a aria-label="Clickable Troubleshooting link" href="http://help.clark.center" target="_blank" rel="noopener noreferrer">Troubleshooting</a>
+        <a [routerLink]="['/editorial-process']" aria-label="Editorial Process" target="_blank" rel="noopener noreferrer">Editorial Process</a>
         <!-- <a aria-label="Clickable Tutorial link" href="http://about.clark.center/tutorial" target="_blank" rel="noopener noreferrer">{{ copy.TUTORIAL }}</a> -->
+        <a [routerLink]="['/about-us']" aria-label="About Us Page link" target="_blank" rel="noopener noreferrer"> About CLARK </a>
+        <a [routerLink]="['/contribute-page']" aria-label="Contribute Page" target="_blank" rel="noopener noreferrer"> Resources for Contributors </a>
```

Comment:
Only if you want to rename this.

```suggestion
        <a [routerLink]="['/contribute']" aria-label="Contribute Page" target="_blank" rel="noopener noreferrer"> Resources for Contributors </a>
```